terraform/Azure/only_private_Blob_storage/only_private_Blob_storage.rego

```diff
@@ -0,0 +1,12 @@
+package torque
```

Comment:
storage VS storage**s** (like 'only_private_S3_bucket**s**')
[dir name + files names]

Reply:
changed to storages